README.md

```diff
@@ -1 +1,85 @@
-# alpha-finance
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+# alpha-finance
+
+This app can be used to make budget, which you can use to make proper budget for any occasion, where that is monthly home budget, married budget or any other occasion.
+
+## Welcome! 👋
+
+## Live Version
+
+[Demo Link](https://alpha-finance.herokuapp.com/) 
+                              
+## Built With ✔️
+
+- Ruby on Rails programming language
+
+## Functionalites 😄
+
+- This is going to be a crud app, in which anyone can register and add catageries and add specific items at those catagories.
+
+## Getting Started 🙌
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  Ruby
+  Rails
+
+```
+### Setup
+Clone the repository and change the directory to portfolio
+
+``` 
+  `$ git clone 'https://github.com/safar1212/alpha-finance'`
+
+  `$ cd alpha-finance`
+
+  `$ bundle install`
+
+  `$ Rails.create()`
+
+  `$ Ruby bin/rails db:migrate`
+
+  `$ http://localhost/3000`
+
+```
+
+### Run tests
+you can run one of the following command in your terminal
+1. Run testing
+```Ruby
+rspec spec
+```
+2. Run linters
+```Ruby
+> npx stylelint "**/*.{css,scss}"
+> Rubocop --color
+> Rubocop -A
+```
+
+
+### Authors
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Safar Ali|[@Safar Ali](https://github.com/safar1212)|[@Safar Ali](https://twitter.com/SafarAli999)|[@Safar Ali](https://www.linkedin.com/in/safar-ali999/)|
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Aknowledgements
+
+Original design idea by Gregoire Vella on Behance
```

Comment:
- Optional
Kindly give the credit to the original designer of the design you have used. I am raising this as optional as the previous reviewer didn't raise this as required.
![image](https://user-images.githubusercontent.com/33268927/195984407-8f7b69aa-7733-423e-b89e-0fe144d9e4e3.png)

Reply:
Oho! Thanks for reminding me of that. I actually forgot that. I will surely do it.

Reply:
@muneebulrehman I am not able to submit the project on the dashboard. I think you've forgotten to toggle that option. Could you please change that? I am still getting the "Review pending" message.

Reply:
@safar1212, I submitted it just after I left the comment here. Are you still having this issue?